test/functional/plugin/shada_spec.lua

```diff
@@ -622,6 +622,7 @@ describe('In autoload/shada.vim', function()
         '  # Required key missing: rc',
```

Comment:
> …
> Also a test where header contains every single key which is defined for any kind of entry.
> And a test where header contains every single key which is defined for any kind of entry, but with value of another type.

Cannot find the third (guess I was wrong: next check looks like it, but has different purpose and less keys), but the second is on line 173.